en/development/debugging.rst

```diff
@@ -51,8 +51,13 @@ Using the Debugger Class
 
 .. php:class:: Debugger
 
-To use the debugger, first ensure that ``Configure::read('debug')`` is
-set to ``true``.
+To use the debugger, in your config/app.php file, ensure that ``'debug' => filter_var(env('DEBUG', true), FILTER_VALIDATE_BOOLEAN),`` is set to ``true``.
```

Comment:
even in the `bootstrap.php` file you can set `Configure::read('debug')` to `true`
and even you can dont have a `config/app.php` 
so , above context 
```
To use the debugger, first ensure that ``Configure::read('debug')`` is
set to ``true``.
```
is more useful , you can keep it as it was ,and add your context after it , 
```
......also you can set `debug` in config/app.php file .......
```